server/views/resources/terms-and-privacy.jade

```diff
@@ -0,0 +1,142 @@
+                    a(href='mailto:team@freecodecamp.com') team@freecodecamp.com
+                    | .
+                p
+                    | All personally identifiable information you provide to us is used by Free Code Camp and its team to process and manage your account, analyze the demographic of our users, or to deliver services through the site.
+                p
+                    | If you choose to provide personally identifiable information to us, you may receive occasional emails from us that are relevant to Free Code Camp, getting a job, or learning to code in general.
+                    | Free Code Camp may also use other third-party providers to facilitate the delivery of the services described above, and these third-party providers may be supplied with or have access to personally identifiable information for the sole purpose of providing these services, to you on behalf of Free Code Camp.
+                p
+                    | Free Code Camp may also disclose personally identifiable information in special legal circumstances. For instance, such information may be used where it is necessary to protect our copyright or intellectual property rights, or if the law requires us to do so.
+                h3
+                    a#Anonymous_Information_15
+                    | Anonymous Information
+                p
+                    | Anonymous aggregated data may be provided to other organizations we associate with for statistical purposes. For example, we may report to an organization that a certain percentage of our site’s visitors are adults between the ages of 25 and 35.
+                h3
+                    a#Cookies_and_BeaconsUse_by_Free_Code_Camp_Opting_Out_18
+                    | Cookies and Beacons—Use by Free Code Camp; Opting Out
+                p
+                    | We use cookies and software logs to monitor the use of
+                    a(href='http://FreeCodeCamp.com') FreeCodeCamp.com
+                    |  and to gather non-personal information about visitors to the site. Cookies are small files that Free Code Camp transfers to the hard drives of visitors for record-keeping purposes. These monitoring systems allow us to track general information about our visitors, such as the type of browsers (for example, Firefox or Internet Explorer), the operating systems (for instance, Windows or Macintosh), or the Internet providers (for instance, Comcast) they use. This information is used for statistical and market research purposes to tailor content to usage patterns and to provide services requested by our customers. To delete these cookies, please see your browser’s privacy settings.
+                p
+                    | A beacon is an electronic file object (typically a transparent image) placed in the code of a Web page. We use third party beacons to monitor the traffic patterns of visitors from one Free Code
+                    a(href='http://Camp.com') Camp.com
```

Comment:
Only the last word is a link, and it's linking to "Camp.com"
![image](https://cloud.githubusercontent.com/assets/553494/12225651/dbee8ca0-b7c3-11e5-8704-508ea0b900a9.png)